data/tutorials/guides/1wf_04_multicore_ready.md

```diff
@@ -211,16 +211,16 @@ WARNING: ThreadSanitizer: data race (pid=26148)
 ```
 
 Notice we obtain a back trace of the two racing accesses, with
-- a write in one `Domain` coming from the array assignment in
-  `Bank.transfer` and
-- a read in another `Domain` coming from a call to
+- A write-in one `Domain`, coming from the array assignment in
```

Comment:
This change alters the meaning AFAICS. The intension was to explain that there was a race between one domain performing a 'write operation' and another domain performing a 'read operation', hence
> - "A write in one `Domain`

Reply:
Thanks for the clarification! I'll change it back.

Reply:
I removed the comma from both lines.

Reply:
Eek, I meant the inserted dash in the `write in` to `write-in` change, but I realize now that that wasn't clear :sweat_smile:

Reply:
OH! So the commas are correct before "coming"?

Reply:
Actually, I like it without the commas there better. 
IRT the write in vs. write-in, I see how I misread it. Thanks for the catch!

Reply:
I'm fine with the latest version, but also admit that I'm probably not the right one to make the call about the comma in the light of all the other ones I missed... :smile: